### PR TITLE
Content ukims number page

### PR DESCRIPTION
**Pull request by besscerule**

### CODE REVIEW

**Review by Ewan-James-Donovan — DISMISSED** [review]
LGTM

**Review by yohan-oc — DISMISSED** [review]

**Review by mycp98 — DISMISSED** [review]

File: app/forms/UkimsNumberFormProvider.scala [hunk 1]
```diff
@@ -26,6 +26,6 @@ class UkimsNumberFormProvider @Inject() extends Mappings {
   def apply(): Form[String] =
     Form(
       "value" -> text("ukimsNumber.error.required")
-        .verifying(maxLength(100, "ukimsNumber.error.length"))
+        .verifying(maxLength(100, "ukimsNumber.error.invalidFormat"))
```
Comment (besscerule) [inline app/forms/UkimsNumberFormProvider.scala]: This line was why it was erroring if we change it back it should work

File: test/forms/UkimsNumberFormProviderSpec.scala [hunk 2]
```diff
@@ -37,13 +37,6 @@ class UkimsNumberFormProviderSpec extends StringFieldBehaviours {
       stringsWithMaxLength(maxLength)
     )
 
-    behave like fieldWithMaxLength(
```
Comment (besscerule) [inline test/forms/UkimsNumberFormProviderSpec.scala]: i think we should keep this test inn and change what was making it fail

Comment (ChrisCookOC) [reply test/forms/UkimsNumberFormProviderSpec.scala]: Happy to do that I just figure we are gonna remove the test later today when we do validation anyway (but also I did think we removed the length validation, didn't realise we kept it!)

File: conf/messages.en [hunk 3]
```diff
@@ -103,9 +103,15 @@ profileSetup.p8 = NIPHL registered traders can add their NIPHL authorisation to
...
-ukimsNumber.error.required = Enter ukimsNumber
+ukimsNumber.title = UK Internal Market Scheme number
+ukimsNumber.h1 = UK Internal Market Scheme number
+ukimsNumber.p1 = HMRC send a letter with a UKIMS authorisation number after you apply. If you are not UKIMS registered, you can not use Trader Goods Profile (TGP).
+ukimsNumber.p2 = A UKIMS number starts with GB or XI, followed by your EORI number, then 14 numbers.
+ukimsNumber.heading = What is your UKIMS number?
+ukimsNumber.hint = For example, XI47699357400020231115081800
+ukimsNumber.linkText = I am not UKIMS registered
+ukimsNumber.error.required = Enter your UKIMS number
 ukimsNumber.error.length = UkimsNumber must be 100 characters or less
```
Comment (ChrisCookOC) [inline conf/messages.en:114]: So is this now unused / deletable?

Comment (besscerule) [reply conf/messages.en:114]: lets get rid

**Review by ChrisCookOC — APPROVED** [review]

**Pull request merged**